build/setup-mac-release-arch.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+set -e
+
+RELEASE_DIR="./dist"
+ARCH=$1
+echo "Setup release metadata for $ARCH architecture"
+
+if [[ "$ARCH" == "arm64" ]]; then
+  ls -t $RELEASE_DIR/latest-mac.yml | head -n 1 | xargs -I {} mv {} $RELEASE_DIR/latest-mac-$ARCH.yml
```

Comment:
this is the same as 
`mv $RELEASE_DIR/latest-mac.yml $RELEASE_DIR/latest-mac-$ARCH.yml`